src/org/opensolaris/opengrok/history/FileHistoryCache.java

```diff
@@ -64,10 +64,11 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(FileHistoryCache.class);
 
     private final Object lock = new Object();
-    private String historyCacheDirName = "historycache";
-    private String latestRevFileName = "OpenGroklatestRev";
+    private final String historyCacheDirName = "historycache";
+    private final String latestRevFileName = "OpenGroklatestRev";
     private boolean historyIndexDone = false;
-    
+    public final static String historyDirName = "historycache";
```

Comment:
I guess the `FileHistoryCacheTest` can live with hardcoded `"historycache"`. If the name is changed the test will fail anyway.